css/main.css

```diff
@@ -106,7 +106,7 @@ body {
     width: min(99%, 90.5rem);
     margin-inline: auto;
     overflow-x: auto;
-    min-height: 95vh;
+    min-height: 93vh;
```

Comment:
This still does not fix the issue of the invisible horizontal scrollbar on my device. I feel like the old percentage value of 85% would be most compatible for min-height instead.